templates/takeovers/_ubuntu-core-18-takeover.html

```diff
@@ -0,0 +1,23 @@
+<section lang="en" data-lang="{{ locale }}" class="p-strip--image is-dark is-deep u-image-position js-takeover p-takeover-ubuntu-core  {% if not default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
```

Comment:
We dont use `data-lang` at all and please remove the `lang` as this takeover is for all